### PR TITLE
improved `setup.py` (and touchup of README.md)

### PR DESCRIPTION
**Pull request by nerohmot**

### CODE REVIEW

File: requirements/run.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 numpy
-spyder >=5
```
Comment (goanpeca) [inline requirements/run.txt]: This should be removed since we are testing directly with master

Comment (nerohmot) [reply requirements/run.txt]: yes, I commented it out (so can be added again when 5 is there)

**Comment by nerohmot** [comment]
what does that mean : nerohmot dismissed goanpeca’s stale review via d3e34f9 3 minutes ago

**Comment by goanpeca** [comment]
You made a push that overwrote my comment maybe?

:-) ?

**Comment by nerohmot** [comment]
🎉 checks are passing!

[d3e34f9](https://github.com/Semi-ATE/Semi-ATE/commit/d3e34f975697ab09145aa4a1f1654d17c667e2d6) is just moving some files ...

**Pull request merged**